client/verta/docs/examples/examples.rst

```diff
@@ -91,4 +75,4 @@ XGBoost
 - `Random Forest with Grid Search
   <https://github.com/VertaAI/modeldb/blob/master/client/workflows/examples/xgboost.ipynb>`__
 - `Client Integration
-  <https://github.com/VertaAI/modeldb/blob/master/client/workflows/examples/xgboost-integration.ipynb>`__
+  <https://github.com/VertaAI/modeldb/blob/master/client/workflows/examples/xgboost-integration.ipynb>`__
```

Comment:
```suggestion
  <https://github.com/VertaAI/modeldb/blob/b536608cd55fe1cd9d8f2cc9bfdbdc092424ae04/client/workflows/examples/xgboost-integration.ipynb>`__
```

less chance of the link going dead in case of restructure, flip side if we change the content of the files the link does not point to latest. No strong preference, either way, feel free to ignore